charts/dvpe-deployment-gloo/values.yaml

```diff
@@ -363,6 +363,12 @@ istio:
 certificate:
       # certificate.useCustomIssuer -- If set to `true` the default issuer's configuration settings will be used. This requires a custom issuer to be installed.
       useCustomIssuer: true
```

Comment:
Is this ever used? We are only providing custom issuers.

Reply:
I can help with that. Some deployments e.g internet facing services don't get certs from the issuer. Another thing is, certificate informations are required although gloo is disabled

Reply:
We could replace this flag with the option `clusterIssuer: none`. Would it be ok?